doc-content/enterprise-only/springboot/spring-boot-persistence-proc.adoc

```diff
@@ -0,0 +1,38 @@
+[id='spring-boot-persistence-proc_{context}']
+= Configuring plugable variable persistence
+
+You can provide an arbitrary entity manager for configured process variable persistence in your {PRODUCT} spring boot application. This is done by adding named beans during the object marshalling strategy resolution. Doing this defines a second entity manager factory based on a qualifier second datasource. This process will not interfere with the primary datasource.
+
+.Prerequisites
+
+* You have a {PRODUCT} Spring Boot project.
```

Comment:
Same as a previous review comment for another topic. Do we need to say "an existing" or something like that about the project prereq?

Reply:
Right!